circleci/images/stylechecker/Dockerfile

```diff
@@ -38,5 +38,5 @@ RUN apk add --no-cache --virtual installdeps \
   && make install \
   && cd ../.. \
   && rm -rf uncrustify-uncrustify-0.68.1/ \
-  && pip3 install -Ir /tmp/etc/requirements.txt \
+  && pip3 install -Ir /tmp/etc/requirements.txt --break-system-packages \
```

Comment:
because this is what we get when building stylechecker image in `pip3 install -Ir /tmp/etc/requirements.txt` stage:

```console
48.03 -- Installing: /usr/local/bin/uncrustify
48.04 -- Installing: /usr/local/share/man/man1/uncrustify.1
50.48 error: externally-managed-environment
50.48 
50.48 × This environment is externally managed
50.48 ╰─> 
50.48     The system-wide python installation should be maintained using the system
50.48     package manager (apk) only.
50.48     
50.48     If the package in question is not packaged already (and hence installable via
50.48     "apk add py3-somepackage"), please consider installing it inside a virtual
50.48     environment, e.g.:
50.48     
50.48     python3 -m venv /path/to/venv
50.48     . /path/to/venv/bin/activate
50.48     pip install mypackage
50.48     
50.48     To exit the virtual environment, run:
50.48     
50.48     deactivate
50.48     
50.48     The virtual environment is not deleted, and can be re-entered by re-sourcing
50.48     the activate file.
50.48     
50.48     To automatically manage virtual environments, consider using pipx (from the
50.48     pipx package).
50.48 
50.48 note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
50.48 hint: See PEP 668 for the detailed specification.
```

Reply:
but should find a better way to workaround that ..